### DISCUSSION TITLE
Timer option: ignore daylight savings for selected timers?

### DISCUSSION THREAD

**Discussion by ajaypala** [arendst/Tasmota#12647] [Q&A] [answered]
Would it be possible to have an option to choose that individual timers use local time without daylight savings (or UTC) instead of local time with day light savings? I need some timers based off local time with daylight savings and some based of local time without daylight savings.

I'm automating come appliances using timers. I want to turn the appliance on/off based on energy costs and my usage habits. My usage habits use local time (with day light savings). My electricity costs change based on local time without daylight savings.

**Comment by sfromis** [comment, marked as answer]
Not a feature of Tasmota.
